app/views/house_members/show.html.erb

```diff
@@ -38,7 +44,7 @@
         <% elsif current_user && !current_user.house_members.include?(@rep)%>
           <%= button_to "Add to Favorites" , "/house_members/#{@rep.id}/favorites", method: :post%>
         <% else %>
-           <p>Click on Register to favorite</p>
+           <p>Log in to add this representative to your dashboard</p>
```

Comment:
I definitely like this wording better, nice fix

Reply:
I agree. This wording is much more clear to the user.